repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/90sys-upgrade/module-setup.sh

```diff
@@ -35,16 +35,23 @@ install() {
     #       won't lose our units after switch-root.
     unitdir="/etc/systemd/system"
 
+    # shellcheck disable=SC2154 # Following variables are assigned by dracut
+    { # NOTE: use these new variables instead to avoid the warning
+    local _initdir=$initdir
+    local _moddir=$moddir
+    local _systemdsystemunitdir=$systemdsystemunitdir
+    }
+
     # Set up systemd target and units
-    upgrade_wantsdir="${initdir}${unitdir}/upgrade.target.wants"
+    upgrade_wantsdir="${_initdir}${unitdir}/upgrade.target.wants"
 
-    inst_simple "$moddir/upgrade.target" "$unitdir/upgrade.target"
+    inst_simple "${_moddir}/upgrade.target" "${unitdir}/upgrade.target"
 
     mkdir -p "$upgrade_wantsdir"
     for s in $upgrade_hooks; do
-        inst_simple "$moddir/$s.service" "$unitdir/$s.service"
-        inst_script "$moddir/$s.sh"      "/bin/$s"
-        ln -sf "../$s.service" $upgrade_wantsdir
+        inst_simple "${_moddir}/${s}.service" "${unitdir}/${s}.service"
+        inst_script "${_moddir}/${s}.sh"      "/bin/$s"
```

Comment:
I am not looking at the consistency across the entire file, however, lets be consistent at line level:
```suggestion
        inst_script "${_moddir}/${s}.sh"      "/bin/${s}"
```